webapi/webapi-nfc-w3c-tests/nfc/support/nfc_common.js

```diff
@@ -29,7 +29,12 @@ Authors:
 
 */
 
-document.write("<script language=\"javascript\" src=\"..\/resources\/testharness.js\"><\/script>");
-document.write("<script language=\"javascript\" src=\"..\/resources\/testharnessreport.js\"><\/script>");
+var head_src = document.head.innerHTML;
+if (head_src.search(/\/testharness.js\W/) === -1) {
+    document.write('<script language="javascript" src="../resources/testharness.js"></script>\n');
```

Comment:
The `language` attribute on `script` has been deprecated in HTML5, see http://www.w3.org/TR/html5-diff/

Moreover, though `type="text/javascript"` is the correct syntax, it overrides the default scripting language. We can also safe leave it out. See
http://stackoverflow.com/questions/2626563/leaving-out-type-text-javascript-language-javascript